tokio/src/sync/mutex.rs

```diff
@@ -219,6 +219,26 @@ impl<T: ?Sized> Mutex<T> {
         }
     }
 
+    /// Creates a new lock in an unlocked state ready for use.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use tokio::sync::Mutex;
+    ///
+    /// static lock: Mutex<i32> = Mutex::const_new(5);
+    /// ```
+    #[cfg(all(feature = "nightly", feature = "parking_lot", not(all(loom, test)),))]
```

Comment:
let's add the `doc_cfg` attr here as well:
```suggestion
    #[cfg(all(feature = "nightly", feature = "parking_lot", not(all(loom, test)),))]
    #[cfg_attr(docsrs, doc(cfg(all(feature = "parking_lot", feature = "nightly"))))]
```